docs/services/sdk/overview.md

```diff
@@ -18,8 +18,50 @@ Add the `@restatedev/restate-sdk` dependency to your NodeJS project to start dev
 The Typescript SDK uses the `protobufjs` and `ts-proto` dependencies to work with the Protobuf-generated Typescript code.
 So add these as well to your project.
 
-## Example of a service
-Below is an example of a Restate Typescript service to get a better understanding of the potential end result.
+## APIs
+Restate currently supports two Typescript APIs for implementing services: the handler API and the gRPC API.
+The gRPC API requires you to specify the service contract (services, methods and data types) via a Protobuf definition.
+The handler API does not require this. 
+
+### Example of a service with the Handler API
+Below is an example of a Typescript service with the Handler API.
```

Comment:
What's the casing for handler API? Here it is capital letter Handler and above is handler.